platform/windows/SCsub

```diff
@@ -60,6 +60,9 @@ sources += res_obj
 prog = env.add_program("#bin/godot", sources, PROGSUFFIX=env["PROGSUFFIX"])
 arrange_program_clean(prog)
 
+if env.msvc:
+    env.Depends(prog, "godot.natvis")
```

Comment:
Technically it's only a dependency with `env["vsproj"]`, no?

Reply:
Actually, no! The file can be used when debugging even without a Visual Studio project, provided it's properly assigned. I debug in VSCode almost exclusively, and it ztill works

Reply:
But it's not compiled into the binary in these cases, is it?
Does this imply that a change to the `.natvis` file means recompiling the whole engine?

Reply:
It recompiles the link step on the executable, so not the whole engine. Truthfully, I'm not sure what the deal with it is, beyond that it needed to be relinked before changes showed up in the debugger

Reply:
Oh right my bad, it _is_ passed to the linker:
```
platform/windows/detect.py
599:    env.Append(LINKFLAGS=["/NATVIS:platform\\windows\\godot.natvis"])
```